recipes/msdf-atlas-gen/all/test_package/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    @ property
+    def _atlas_texture_file(self):
+        return os.path.join(self.build_folder, "atlas_texture.png")
+
+    @ property
```

Comment:
`tools.cross_building` compares with the settings of the conan object.
```suggestion
class TestPackageConan(ConanFile):
    settings = "os", "arch", "compiler", "build_type"

    @property
    def _atlas_texture_file(self):
        return os.path.join(self.build_folder, "atlas_texture.png")

    @property
```

Reply:
Thank you, I would have struggled finding what was the issue here